### DISCUSSION TITLE
not work

### DISCUSSION THREAD

**Discussion by netwons** [projectdiscovery/shuffledns#99] [Q&A] [answered]
![This is an image](<a href="https://ufile.io/9gqte4cc" target="_blank" title="Download from https://ufile.io/ for free">Download پروژه_ی جدید.png from https://ufile.io/ for free</span></a>)

this code not work but in version 1.0.5 worked

**Comment by ehsandeep** [comment]
@netwons can you include the error details?

**Reply by netwons** [reply, marked as answer]
not error
not work
but install old version .its okey & worked
